src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_table/table.tsx

```diff
@@ -10,20 +10,13 @@
 import React, { useCallback, useMemo, useState } from 'react';
 import useWindowSize from 'react-use/lib/useWindowSize';
 import useLocalStorage from 'react-use/lib/useLocalStorage';
-import type {
-  EuiDataGridProps,
-  EuiDataGridCellPopoverElementProps,
-  EuiSwitchEvent,
-} from '@elastic/eui';
+import type { EuiSwitchEvent } from '@elastic/eui';
```

Comment:
Do you think we could rename this file now? Maybe something more specific of what it is? Something like `table_with_actions` or something else?